### PR TITLE
adds info about October call to Community Call landing page

### LINKED ISSUE #2123: Add Oct call info to Comm Call page
From @KGorr:

Hi @hannahkane, is it too late to also request that the following info be added to the "upcoming workshops" section at the bottom of the page?

October 13
Ada Lovelace Day
Join the Mozilla Learning and Mozilla Science teams to celebrate Ada Lovelace Day and women around the globe that are making a difference in science, web literacy and beyond.

@ldecoursy @auremoser

### PR DESCRIPTION
**Pull request by hannahkane**
- [X ] Localization addressed (Not Applicable)

Fixes #2123

**Changes proposed in this pull request:**
- adds copy about October community call to landing page

### CODE REVIEW

File: pages/community-call.jsx [hunk 1]
```diff
@@ -58,8 +58,11 @@ var CommunityCallPage = React.createClass({
 
           <ul className="upcoming-workshops">
             <li>
-              <p className="date">October TBD</p>
-              <h2>Theme: TBD</h2>
+              <p className="date">October 13, 2016</p>
+              <h2>Ada Lovelace Day</h2>
+              <p>
+                Join the Mozilla Learning and Mozilla Science teams to celebrate Ada Lovelace Day and women around the globe who are making a difference in science, web literacy and beyond.
```
Comment (Pomax) [inline pages/community-call.jsx]: is http://findingada.com/ a thing we want to link to for the "Ada Lovelace Day" bit?

Comment (hannahkane) [inline pages/community-call.jsx]: @KGorr - thoughts about Pomax's suggestion above?

Comment (KGorr) [inline pages/community-call.jsx]: Seems ok to me, but would love @auremoser's input here too.

Comment (auremoser) [inline pages/community-call.jsx]: Yes, sounds great. 👍 Thank you!

Comment (auremoser) [inline pages/community-call.jsx]: As an aside, here's our [post](https://science.mozilla.org/programs/events/community-call-oct-13-11-et) feel free to cross-link! Would love to link to yours as well!

**Comment by hannahkane** [comment]
added link. @Pomax - is this good to go?

**Comment by Pomax** [comment]
looks good to me!

**Pull request merged**